### ISSUE TITLE
Add discussion about zoneinfo package to wasm-constraints.md

### ISSUE THREAD

**Issue by hoodmane** [pyodide/pyodide#4668]
## 📚 Documentation

By default, `zoneinfo` is missing all the timezones. It's necessary to install the `tzdata` package to get them. I think we should:
1. mention this in wasm-constraints docs
2. add tzdata package recipe (I thought we already had one but it doesn't seem so)
3. maybe make tzinfo warn instructing the user to install `tzdata` if it's used without installing `tzdata`

**Comment by joemarshall** [comment]
yes to this. Needed for pyarrow also, so I don't mind doing it while I'm on that. It's a pure python package, so the recipe should be trivial.

**Comment by hoodmane** [comment]
Yeah I don't think we want to add 300kb. People can install tzdata if they need it. Ideally we ought to put a custom warning if they import zoneinfo without loading tzdata though.

**Comment by joemarshall** [comment]
> Yeah I don't think we want to add 300kb. People can install tzdata if they need it. Ideally we ought to put a custom warning if they import zoneinfo without loading tzdata though.

Yeah, I added a patch to give a warning when you lookup a timezone with zoneinfo and it fails. That way it doesn't error if you don't need to see the error (e.g. you're including some package that imports zoneinfo but don't use any time zone related functionality yourself), and also it doesn't error if you're in node.js with access to /usr/local/zoneinfo )
